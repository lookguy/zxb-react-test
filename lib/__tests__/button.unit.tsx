import Button from "../button";
import renderer from 'react-test-renderer'
import React from 'react';

describe('button tests', () => {
    it('是个div', () => {
        // const tree = renderer
        //     .create(<Button icon="alipay"><span>Click Me</span></Button>)
        //     .toJSON()
        // expect(tree).toMatchSnapshot()
        const json = renderer.create(<Button></Button>).toJSON()
        expect(json).toMatchSnapshot()
    })
})