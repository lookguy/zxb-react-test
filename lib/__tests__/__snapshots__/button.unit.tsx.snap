// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button tests 是个div 1`] = `
<div>
  按钮
</div>
`;
